src/components/pages/AnnotationPage.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component}  from "react";
+import { connect } from "react-redux";
+import { Link, useParams } from "react-router-dom";
+import { fetchAnnotationDetails } from "../../actions";
+import ImageAnnotation from "../ImageAnnotation";
+
+function AnnotationPage (props) {
```

Comment:
Added annotation page that wraps image annotation providing image src url to the image annotation component